setup.py

```diff
@@ -68,9 +68,11 @@ def read_file(filename, encoding='utf8'):
 ]
 
 docs_extras = [
+    # Bug in Sphinx < 1.5.1
+    # https://github.com/sphinx-doc/sphinx/issues/3212
+    'docutils<0.13.1',
```

Comment:
I don't think we should require an older version of docutils here. Newer versions of Sphinx work just fine with newer versions of docutils and this will cause problems on up to date distros that only have newer versions of both packages available. Really what we want to say is if you are using an older version of Sphinx then we require an older version of docutils, but I don't think there's a good way to do that.

In general, this is a bit of an ugly problem that we've hit before with other dependencies. In those cases, we've done nothing and left it up to the user and/or their package manager/maintainers to ensure compatible versions of our dependencies are installed. This certainly isn't elegant, but it hasn't caused any issues. Distro package maintainers (usually) test packages before pushing them live and we wouldn't be the only Python package that would have issues here. For people installing from `pip`, `pip` prefers the latest version of everything which usually has all the latest bug fixes.

It's probably a good idea to mention in a comment here that `docutils<0.13.1` is required when used with `Sphinx<1.5.1` and provide a link to the GitHub issue, but I think the actual dependency should be on an unpinned back version of docutils.